CHANGELOG.md

```diff
@@ -1,3 +1,143 @@
+# 84.0.0
+
+In this release we corrected a text field accesssibility behavior added ripple to chips and added
+additional delegate calls to post bottom sheet's scrolling.
+
+We finished updating additional theming docs for multiple components including chips, dialogs, cards.
```

Comment:
> We finished updating additional theming docs for multiple components including Chips, Dialogs, Cards.

Reply:
Should we put an "and" before Cards as well?